templates/vue/src/components/TapestryAccordion.vue

```diff
@@ -6,45 +6,53 @@ export default {
       type: Array,
       required: true,
     },
-    defaultIndex: {
+    nodeId: {
       type: Number,
       required: false,
       default: 0,
     },
   },
-  data() {
-    return {
-      activeRow: null,
-    }
-  },
   computed: {
     hasNext() {
       return this.rows.indexOf(this.activeRow) < this.rows.length - 1
     },
-  },
-  created() {
-    if (this.defaultIndex >= 0) {
-      const row = this.rows[this.defaultIndex]
-      if (row) {
-        this.activeRow = row
-      }
-    }
+    activeRowIndex() {
```

Comment:
I think it would be OK to see the route changes there as well, as long as we also see the route changes for everything else in the spaceship cockpit so that the URL can take the user to a specific module / subtopic / accordion row in the cockpit. It's OK with me as long as it behaves in a way that the user doesn't get surprised and that it's consistent, so I'll leave it up to you to make that decision.

Reply:
I've added a prop to the component to allow for the route changes to remain constant on TYDE as well. When we port the feature, it should behave as expected which I believe is OK.